can/interfaces/__init__.py

```diff
@@ -32,14 +32,16 @@
 try:
     from importlib.metadata import entry_points
 
-    entry = entry_points()
-    if "can.interface" in entry:
-        BACKENDS.update(
-            {
-                interface.name: tuple(interface.value.split(":"))
-                for interface in entry["can.interface"]
-            }
-        )
+    try:
+        entries = entry_points(group="can.interface")
+    except TypeError:
+        # Fallback for Python <3.10
+        # See https://docs.python.org/3/library/importlib.metadata.html#entry-points, "Compatibility Note"
```

Comment:
We could also use `importlib_metadata` from PyPI as fallback

Reply:
That would also be possible, yep.
As it's just so little effort, I thought it's simpler to just handle the fallback here.